frontend/src/components/explorer/gemBrowser/GemSearch.vue

```diff
@@ -292,23 +290,21 @@ export default {
       display: none;
     }
 
-    .searchResultSection{
-      > a {
+    .searchResultSection {
+      > div {
         display: block;
         padding: 7px 0px;
         white-space: nowrap;
         overflow: hidden;
         text-overflow: ellipsis;
       }
 
-      .search-result-icons a {
-        &:first-child {
-          color: $primary;
-        }
+      .color-gb {
+        color: $primary;
+      }
 
-        &:nth-child(2) {
-          color: #A15786;
-        }
+      .color-ip {
+        color: #A15786;
```

Comment:
Since this color is shared with `GemSearch.vue`, maybe it would make sense to define it in the CSS vars.

Reply:
> Since this color is shared with `GemSearch.vue`, maybe it would make sense to define it in the CSS vars.

Do you mean define it under the root of `style` in this file? It seems this coloring, i.e. `$primary` for the icon Gem-Browser and `#A15786` for the icon IP is set only at this place and at `InteractionPartners.vue`. All the rest places, such as the Map Viewer and Documentation, another set of colors are used. Should we make them consistent? As I can see, current coloring at this search window looks quite nice and I guess there was some effort to define these colors when this feature was introduced.

Reply:
Most (all?) colors used in the codebase are defined in this file https://github.com/MetabolicAtlas/MetabolicAtlas/blob/develop/frontend/src/style/vars.scss

Searching for `#A15786` yields 2 matches in the codebase, in `GemSearch.vue` and `InteractionPartners.vue`. So instead of defining the same value, it would make sense to me to make this color available in the entire codebase by adding it to `vars.scss`.

Reply:
Make sense to me. I will also leave a note in the private issue [#84](https://app.zenhub.com/workspaces/metatlas-roadmap-60d96b68da0fb10019d26deb/issues/metabolicatlas/private-issues/84) for refactoring of the color definition for IP.

Reply:
refactored with the commit 92dda32